subtitles/zh-CN/21_preprocessing-sentence-pairs-(pytorch).srt

```diff
@@ -115,12 +115,12 @@ implies the second.
 
 24
 00:01:06,930 --> 00:01:08,820
-所以分类成对的句子是一个问题
+所以分类成对的句子是值得研究的问题。
 So classifying pairs of sentences is a problem
 
 25
 00:01:08,820 --> 00:01:10,260
-值得研究。
+

```

Comment:
```suggestion
所以分类成对的句子是值得研究的问题。
```

In this case, to keep a better caption experience, please copy the whole sentences to the next subtitle line.